testing/server.go

```diff
@@ -4,6 +4,7 @@
 package testing
```

Comment:
I have to say I'm not so keen on yet another "testing" package - there are dozens
in juju-core and it can be confusing. I'd be inclined to name it "charmstoretesting"
because most importers will want to rename it to something like that anyway.